sqe-database-access/Models/UserModels.cs

```diff
@@ -36,6 +36,17 @@ public class UserEditionPermissions
         public bool Locked { get; set; }
     }
 
+    public class AddEditorJWT
```

Comment:
If this is stored in the token, and nowhere else, I don't think it belongs in the database-access assembly. It's going to be changed into an actual model that is going to be saved in the database, so this is just a bit of nitpicking.